CHANGELOG.md

```diff
@@ -40,7 +40,7 @@
 - PR #1354 Fix `fillna()` behaviour when replacing values with different dtypes
 - PR #1347 Fixed core dump issue while passing dict_dtypes without column names in `cudf.read_csv()`
 - PR #1379 Fixed build failure caused due to error: 'col_dtype' may be used uninitialized
-
+- PR #1395 Update CONTRIBUTING to use the environment variable CUDF_HOME
```

Comment:
```suggestion
- PR #1395 Update CONTRIBUTING to use the environment variable CUDF_HOME

```

Reply:
Fixed. Also fixing the fact that there is only one line break between the 0.5.1 and 0.6 versions.